lib/core/src/sdk.rs

```diff
@@ -539,6 +559,11 @@ impl LiquidSdk {
                 handle,
             });
         }
+        for plugin in &self.plugins {
+            plugin
+                .on_start(self.clone(), PluginStorage::new(self.persister.clone()))
```

Comment:
This passes the same storage to all plugins allowing them to override each other's keys. 
If the PluginStorage is also passed with some identifier (plugin id?) we can add this prefix to every key in the implementation.